setup.cfg

```diff
@@ -65,7 +65,7 @@ tests =
     flake8-2020
     virtualenv>=13.0.0
     pytest-virtualenv>=1.2.7
-    pytest>=3.7
+    pytest>=3.7,<6
```

Comment:
Should you maybe open an issue to make tests pass for pytest >= 6 if there is already not one?

Reply:
I think https://github.com/pypa/setuptools/pull/2487 would fix but that hasn't been reviewed yet.  I reverted my change here.